README.md

```diff
@@ -131,6 +122,6 @@ v4.0.0
 
 See `library.json` (PlatformIO) or `library.properties` (Arduino).
 
-[Project OWL]: <https://www.project-owl.com/>
-[ClusterDuck Protocol]: <https://github.com/Call-for-Code/ClusterDuck-Protocol/wiki>
-[Slack Workspace]: <https://www.project-owl.com/slack>
+[OWL Integrations]: <https://www.owlintegrations.com/>
+[ClusterDuck Protocol]: <https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/wiki>
+[Slack Workspace]: <https://www.project-owl.com/slack>
```

Comment:
Need to update to Discord and remove Slack

Reply:
updated!